language/diem-framework/modules/doc/CoreAddresses.md

```diff
@@ -5,17 +5,12 @@
 
 Module providing well-known addresses and related logic.
 
-> Note: this module currently defines zero-argument functions like <code><a href="CoreAddresses.md#0x1_CoreAddresses_DIEM_ROOT_ADDRESS">Self::DIEM_ROOT_ADDRESS</a>()</code> using capitalization
```

Comment:
Can you possibly add support for named addresses to the documentation generator? Or is this not part of the move-model yet?

Reply:
I'll see what I can do